packages/@glimmer-workspace/integration-node-tests/test/prettier.test.ts

```diff
@@ -0,0 +1,207 @@
+   *  4. pack and add the tarballs to the project
+   *  5. See if a basic import and call to pre-process works
+   */
+  it('Uses the real build, and not our monorepo infra', { timeout: 20_000 }, async () => {
+    /**
+     * pnpm packs tgz's with the name format ${hyphenated-package-name}-${version}.tgz
+     */
+    async function versionOf(name) {
+      let manifestPath = join(monorepoRoot, 'packages', name, 'package.json');
+      let manifestString = await readFile(manifestPath);
+      return JSON.parse(manifestString.toString()).version;
+    }
+
+    let tarballs = {
+      syntax: `glimmer-syntax-${await versionOf('@glimmer/syntax')}`,
+      util: `glimmer-util-${await versionOf('@glimmer/util')}`,
+      wireFormat: `glimmer-wire-format-${await versionOf('@glimmer/wire-format')}`,
+    };
+
+    let file = `console.log((await import('@glimmer/syntax')).preprocess('<h1></h1>'));`;
+    let manifest = JSON.stringify({
+      name: 'real-test',
+      type: 'module',
+      private: true,
+      devDependencies: {
+        '@glimmer/syntax': `file://./${tarballs.syntax}`,
+        '@glimmer/util': `file://./${tarballs.util}`,
+        '@glimmer/wire-format': `file://./${tarballs.wireFormat}`,
+      },
+      pnpm: {
+        overrides: {
+          '@glimmer/syntax': `file://./${tarballs.syntax}.tgz`,
+          '@glimmer/util': `file://./${tarballs.util}.tgz`,
+          '@glimmer/wire-format': `file://./${tarballs.wireFormat}.tgz`,
+        },
+      },
+    });
+
+    async function newTmpDir() {
+      const tmpDir = await mkdtemp(
+        join(os.tmpdir(), `glimmer-node-integration-testing-${Date.now()}-`)
+      );
+
+      return tmpDir;
+    }
+
+    function inDir(dir: string, cmd: string, options = {}) {
+      return $({ cwd: dir, preferLocal: true, shell: true, ...options })(cmd);
+    }
+    function inRoot(cmd: string) {
+      return inDir(monorepoRoot, cmd);
+    }
+
+    function inTmp(cmd: string, options = {}) {
+      return inDir(tmp, cmd, options);
+    }
+
+    //////////
+    // 1 build
+    //   When turbo is set up with "dependsOn": "["^prepack"], we see these packages
+    //   - @glimmer/syntax
+    //   - @glimmer/util
+    //   - @glimmer/wire-format
+    //
+    //   So these 3 packages need to be packed and installed
+    await inRoot(`pnpm turbo --filter "@glimmer/syntax" prepack`);
+
+    //////////
+    // 2 create a space that doesn't mess up the repo
+    let tmp = await newTmpDir();
+
+    // eslint-disable-next-line no-console
+    console.debug(`Project can be inspected at ${tmp}`);
+
+    //////////
+    // 3 create a project that represents real consumer usage
+    await writeFile(join(tmp, 'package.json'), manifest);
+    await writeFile(join(tmp, 'index.js'), file);
+
+    //////////
+    // 4 install the tarballs using stable names so we don't have to
+    //   dynamically build the package.json
+    let packToTemp = `pnpm pack --pack-destination ${tmp}`;
```

Comment:
Yeah, I see. This is why I bumped to `pnpm@10`. It's [not documented](https://pnpm.io/cli/pack), but they [added an `--out` flag](https://github.com/pnpm/pnpm/pull/8900) in pnpm 10, which makes it possible to pack directly into the exact file name you want without hassle.